app/views/spina/admin/pages/_form.html.erb

```diff
@@ -4,67 +4,36 @@
       <% if @page.draft? %>
         <span class="text-gray-400 ml-2 text-sm">(<%= Spina::Page.human_attribute_name(:draft) %>)</span>
       <% end %>
-      
       <%= link_to @page.materialized_path, class: 'px-3 py-2 flex items-center text-gray-400 hover:text-gray-700', target: :blank, data: {turbo: false} do %>
         <%= heroicon("external-link", style: :solid, class: "w-4 h-4") %>
       <% end %>
     <% end %>
-    
     <% header.actions do %>
       <%= render Spina::Pages::ActionsComponent.new(page: @page, locale: @locale) %>
     <% end %>
-    
     <% header.navigation do %>
       <nav class="-mb-3 mt-4">
         <ul class="inline-flex w-auto rounded-md bg-white">
-          <button type="button" class="block px-3 leading-relaxed py-1 hover:text-gray-800 rounded-md text-gray-400 font-medium text-sm flex items-center whitespace-nowrap" data-action="tabs#show" data-tabs-target="button" data-pane-id="page_content">
-            <%= heroicon('document-text', class: 'w-4 h-4 mr-1 -ml-1 opacity-75') %>
-            <%=t 'spina.pages.page_content' %>
-          </button>
-          <button type="button" class="block px-3 leading-relaxed py-1 rounded-md hover:text-gray-800 text-gray-400 font-medium text-sm flex items-center whitespace-nowrap" data-action="tabs#show" data-tabs-target="button" data-pane-id="seo">
-            <%= heroicon('search', style: :solid, class: 'w-4 h-4 mr-1 -ml-1 opacity-75') %>
-            <span class="hidden md:inline">
-              <%=t 'spina.pages.search_engines' %>
-            </span>
-            <span class="md:hidden">
-              SEO
-            </span>
```

Comment:
This markup was used to show a shorthand in the mobile view. We can let the `TabButtonComponent` render just the `<button>` element, and keep this in when we need it.

Reply:
Readded this with my last commit.